src/components/content/reports/ReportsServiceError.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert } from 'antd';
+import React from 'react';
+import { ApiError, Response } from '../../../xpanse-api/generated';
+import { convertStringArrayToUnorderedList } from '../../utils/generateUnorderedList';
+
+export default function ReportsServiceError({ error }: { error: unknown }): React.JSX.Element {
```

Comment:
this component is duplicate of MyServicesError. We must reuse code of existing components here.